src/tools/ceph_kvstore_tool.cc

```diff
@@ -90,8 +90,7 @@ int main(int argc, const char *argv[])
   string path(args[1]);
   string cmd(args[2]);
 
-  if (type != "leveldb" &&
```

Comment:
For a second I was thinking about a check for the `type == "leveldb"` to say it's out but after another one I doubted it makes sense. Thoughts?

Reply:
In the help message, there is no leveldb, leveldb is not supported any more, i think is no need to check `type == "leveldb"`

Reply:
I see your point. I was thinking about differentiating the error message: "Unrecognized type" vs "Removed type" but I'm fine with leaving as is.